templates/quickstart-crowd-dc-with-vpc.template.yaml

```diff
@@ -344,7 +341,7 @@ Parameters:
     ConstraintDescription: Must be 'Amazon Aurora PostgreSQL' or 'PostgreSQL'.
     Type: String
   DBInstanceClass:
-    Default: db.t2.medium
+    Default: db.m4.large
```

Comment:
As used by Confluence and Bitbucket

Reply:
Hmm, I feel like we should be defaulting to the newest generation of instance class. Perhaps Confluence and Bitbucket need to be changed. I can't imagine there would be a negative effect and I think it is best practice to use the newer instances. What do you think @nanux @dylanrathbone

Reply:
Unfortunately my knowledge on these instance classes is limited particularity with regards Crowds DB requirements. I raised this on a call I had with the Crowd team last night, which we are going to work through. Details on this subject here: https://hello.atlassian.net/wiki/spaces/DC/pages/669394104/AWS+Instance+sizing